PAbishai/Ride-My-Way
# #158636944 Create unit tests for API endpoints

**What does this PR do**
Make tests for API endpoints

**Description of tasks to be completed?**
Get the below tests to pass
test_ POST /api/v2/auth/signup 
test_ POST /api/v2/auth/login
test_ POST /api/v2/users/rides

**How should this be manually tested?**
Install postgress
Create a database ride-my-way
`createdb ride-my-way`
Create tables users and rides
```
`CREATE TABLE rides (
id SERIAL PRIMARY KEY,
user_id  INT NOT NULL,
location VARCHAR NOT NULL,
destination VARCHAR NOT NULL,
leaving VARCHAR NOT NULL, passengers VARCHAR)
;
```
```
CREATE TABLE users (
id SERIAL PRIMARY KEY,
name varchar NOT NULL,
email varchar NOT NULL,
dl_path varchar,
car_reg varchar,
password varchar
);
```
Clone the repo
Create a virtualenv 
`#virtualenv Ride-My-Way`
cd into the repo and activate it
run
`pip install -r requirements.txt`
cd into the resources folder
edit 'api_v2_users.py'
add your postgres username and password to conn_string
cd out of resources and into tests
run 
`pytest test_apis_v2.py`
the tests should pass

**What are the relevant pivotal tracker stories**
[#158636944](https://www.pivotaltracker.com/story/show/158636944)
